src/EffectPlugin.cpp

```diff
@@ -25,9 +25,9 @@ bool EffectInstanceEx::Init()
 }
 
 EffectUIValidator::EffectUIValidator(
-   EffectUIClientInterface &effect, EffectSettingsAccess &access)
-   : mEffect{effect}
-   , mAccess{access}
+   EffectUIServices &services, EffectSettingsAccess &access)
+   : mServices{ services }
```

Comment:
Minor suggestion - let's call it mUIServices maybe?

Reply:
Ok